packages/rspack-cli/src/utils/profile.ts

```diff
@@ -0,0 +1,83 @@
+import assert from "assert";
+import inspector from "inspector";
+import fs from "fs";
+
+export type JavascriptCPUProfile = {
+	kind: "js cpu profile";
+	output: string;
+};
+export type RustTrace = {
+	kind: "rust trace";
+	filter: string;
+	output: string;
+};
+
+export type ProfileOptions = JavascriptCPUProfile | RustTrace;
+
+const defaultRustTraceOutput = `./rspack.trace`;
```

Comment:
should we put it into cwd or in the output_dir? nextjs seems put it into output_dir

Reply:
current is cwd